### DISCUSSION TITLE
FreeCodeCamp and CSS Diner feedback

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#83] [Week 3 - Instructional Pros and Cons]
The HTML and Javascripts exercises this week were a bit more helpful and I got more out of them compared to the exercises last week. I'm not sure if I am just more comfortable with the languages or if they were just designed better. I'm leaning toward that they were designed better. 
The freecodecamp is difficult for me just because of the lack of tutorials and the hints do not help me very much, I usually have to spend a lot of time on Youtube going over how to perform certain tasks. It would be nice especially for novices like me to have more step-by-step guides associated with the exercises. 
The CSS Diner I actually really enjoyed! This was a fun way to learn more functions and experience the code. I like how it was more interactive and gave some feedback and help.

**Comment by metasean** [comment]
> I'm not sure if I am just more comfortable with the languages or if they were just designed better. 

Many people have a proclivity for HTML and CSS, _or_ JavaScript.  These platforms tend to mirror these tendencies.  For, example, there are folks who think FreeCodeCamp exercises are well defined and provide lots of feedback, while they struggle with CSS Diner.  Odds are, you have a natural tendency toward HTML & CSS, while JavaScript is likely to be more challenging for you.
